systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -328,10 +334,10 @@ private static String getOrDefault(String varName, String defaultValue) {
 
     private static <T> T getOrDefault(String var, Function<String, T> converter, T defaultValue) {
         String value = isEnvVarSet(var) ?
-                System.getenv(var) :
-                (Objects.requireNonNull(YAML_DATA).get(var) != null ?
-                        YAML_DATA.get(var).toString() :
-                        null);
+            System.getenv(var) :
```

Comment:
Yoi can use Environment from TF so no need to have getordefault etc

Reply:
I'm doing this piece by piece, not everything in one PR.
It's just that I copied stuff from other place so it just changed some of the indents etc. (as you can anyway see).
Plus, the PR is not opened, still in draft, so I expect that I will anyway change it :)